src/gobimport/__main__.py

```diff
@@ -2,17 +2,45 @@
 
 This component imports data sources
 """
+from gobcore.exceptions import GOBException
 from gobcore.message_broker.config import WORKFLOW_EXCHANGE, IMPORT_QUEUE, RESULT_QUEUE
 from gobcore.message_broker.messagedriven_service import messagedriven_service
 
 from gobimport.import_client import ImportClient
-from gobimport.mapping import get_mapping
+from gobimport.mapping import get_dataset_file_location, get_mapping
 
 
-def handle_import_msg(msg):
-    dataset_file = msg['dataset_file'] if 'dataset_file' in msg else msg.get('header').get('dataset_file')
-    assert dataset_file
+def extract_dataset_from_msg(msg):
+    """Returns location of dataset file from msg.
+
+    Dataset files can be identified by the file location ('dataset_file') or dataset identifier ('dataset') in the root
+    of the message or in the 'contents' part of the message:
+
+    message = { "dataset_file": "data/somedataset.json" }
```

Comment:
Why not supply for message contents like: {
'catalogue': 'gebieden',
'collection': 'stadsdelen',
'application': 'DGDialog' } which then maps to 'data/stadsdelen.json'
Import is then made explicit, it does exactly what is stated in the attributes.

Reply:
We should also change the import call from GOB-Workflow to match this. It would be nice since all commands are then called in the same manner:
gobworkflow.start import catalogue collection
gobworkflow.start export catalogue collection
gobworkflow.start relate catalogue

Reply:
The catalogue collection may have a default application (e.g. DGDialog for gebieden) but can be overridden, e.g. by setting it to DIVA.
And you can always specify only a dataset, e.g. data/test.json